### PR TITLE
[AAR-2022] Add organiser photos

### PR DESCRIPTION
**Pull request by MadsBaggesen**
Updating a sponsorlogo (from my own company) and adding one more organizer profile image.

### CODE REVIEW

**Comment by yvovandoorn** [comment]
Hey @MadsBaggesen as this includes a new logo for eficode, could you please follow https://github.com/devopsdays/devopsdays-web/blob/main/utilities/README.md#updating-a-sponsor-image-or-data. 

Thanks!

**Comment by phrawzty** [comment]
@MadsBaggesen As a general policy, we prefer not to rewrite history, especially as it pertains to assets used by multiple locales. Please use the sponsor update tool. Thank you.

**Comment by MadsBaggesen** [comment]
When I run the script in Git Bash on windows, I get an `sed` error. Is it supposed to work on Git Bash on windows, or do I need to find a linux bash shell to execute the script?

Mads Baggesen@DESKTOP-OKMJEU4 MINGW64 ~/Documents/devopsdays/devopsdays-web (2022-AAR-update-website-for-aarhus)
$ ./utilities/change_sponsor_logo.sh eficode ~/Downloads/eficode.png
Modifying 2019-amsterdam.yml
sed: can't read s/id:[ ]+eficode[ $]*/id: eficode-before-20211021/: No such file or directory

Mads Baggesen@DESKTOP-OKMJEU4 MINGW64 ~/Documents/devopsdays/devopsdays-web (2022-AAR-update-website-for-aarhus)

File: data/events/2022-aarhus.yml [hunk 1]
```diff
@@ -83,10 +83,10 @@ team_members: # Name is the only required field for team members.
     image: "johanne-sundtoft.jpg"
   - name: "Peter Andersson"
     employer: "Eficode"
-#    image: "peter-andersson.jpg"
+    image: "peter-andersson.jpg"
   - name: "Jonas Vinther"
     employer: "Trifork"
-#   image: "jonas-vinther.jpg"
+    image: "jonas-vinther.jpg"
```
Comment (phrawzty) [inline data/events/2022-aarhus.yml:89]: `danny-lajer.jpg` already exists, and you add `peter-andersson.jpg` in this PR; _however_, `jonas-vinther.jpg` neither exists nor is it added in this PR.

**Pull request merged**